lib/vanagon/platform.rb

```diff
@@ -221,10 +221,19 @@ def is_cisco_wrlinux?
       return !!@name.match(/^cisco-wrlinux-.*$/)
     end
 
-    # Utility matcher to determine is the platform is an osx variety
+    # Utility matcher to determine if the platform is an osx variety
     #
+    # @deprecated Please use is_macos? instead
     # @return [true, false] true if it is an osx variety, false otherwise
     def is_osx?
+      warn "is_osx? is a deprecated method, please use is_macos? instead."
+      is_macos?
+    end
+
+    # Utility matcher to determine if the platform is a macos variety
+    #
+    # @return [true, false] true if it is a macos variety, false otherwise
+    def is_macos?
       return !!@name.match(/^osx-.*$/)
```

Comment:
Thinking about this, we might also want to add a matcher on `macos`; we should preserve old behavior and add new behavior.